### PR TITLE
LPS-121920

### PR DESCRIPTION
**Pull request by jonmak08**
Hey @liferay-echo,

Attached is an update for http://issues.liferay.com/browse/LPS-121920.

Test plan:
- Add web content to Liferay
- Delete web content to Recycle Bin
- Delete/Restore content in Recycle Bin

trash

Please let me know if you have any questions. Thanks!

### CODE REVIEW

**Comment by jonmak08** [comment]
ci:test

**Comment by jonmak08** [comment]
ci:test:sf

**Comment by jonmak08** [comment]
ci:test:relevant

File: modules/apps/trash/trash-web/src/main/resources/META-INF/resources/js/TrashManagementToolbarPropsTransformer.js [hunk 1]
```diff
@@ -12,27 +12,39 @@
...
 
-export default ManagementToolbarDefaultEventHandler;
+	return {
+		...otherProps,
+		onActionButtonClick: (event, {item}) => {
+			const action = item?.data?.action;
+
+			if (action === 'deleteSelectedEntries') {
+				deleteSelectedEntries();
+			} else if (action === 'restoreSelectedEntries') {
```
Comment (markocikos) [inline modules/apps/trash/trash-web/src/main/resources/META-INF/resources/js/TrashManagementToolbarPropsTransformer.js:45]: Minor SF:
```
}
else if
```

File: modules/apps/trash/trash-web/src/main/resources/META-INF/resources/js/TrashManagementToolbarPropsTransformer.js [hunk 2]
```diff
@@ -12,27 +12,39 @@
...
+	const deleteSelectedEntries = () => {
 		if (
 			confirm(
 				Liferay.Language.get('are-you-sure-you-want-to-delete-this')
 			)
 		) {
-			submitForm(this.one('#fm'));
+			const form = document[`${portletNamespace}fm`];
+
+			submitForm(form);
```
Comment (markocikos) [inline modules/apps/trash/trash-web/src/main/resources/META-INF/resources/js/TrashManagementToolbarPropsTransformer.js:28]: Minor SF: no need for var, can be inline. The same for `form` in `restoreSelectedEntries`.

**Review by markocikos — CHANGES_REQUESTED** [review]
Apart from minor SF changes, LGTM!

Unique CI failure looks unrelated.

**Comment by ealonso** [comment]
Hey @jonmak08,
Could you please take a look to sf error and @markocikos comments?
Regards,
Eudaldo.

**Comment by jonmak08** [comment]
Hey @ealonso and @victorg1991,

I have updated this pull request with the SF changes @markocikos requested.  Please review and let me know if you have any other questions or updates.

Thanks!

**Comment by jonmak08** [comment]
ci:test:sf

**Comment by jonmak08** [comment]
ci:test:relevant

**Review by markocikos — APPROVED** [review]

**Review by julien — APPROVED** [review]

**Comment by ealonso** [comment]
ci:forward

**Pull request closed without merging**